packages/spec/integration/typescript-styled-components/index.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import { render } from 'hops';
+import {} from 'styled-components/cssprop';
```

Comment:
What is this wizardry? If you want to import it as a side-effect, why don't you do `import 'styled-components/cssprop';`?

Reply:
https://styled-components.com/docs/api#usage-with-typescript 😢